pkg/cmd/ssh/options.go

```diff
@@ -1103,6 +1141,15 @@ func getNodes(ctx context.Context, c client.Client) ([]corev1.Node, error) {
 	return nodeList.Items, nil
 }
 
+func getMachines(ctx context.Context, name string, c client.Client) ([]machinev1alpha1.Machine, error) {
```

Comment:
```suggestion
func getMachines(ctx context.Context, c client.Client, namespace string) ([]machinev1alpha1.Machine, error) {
```
rename name to namespace and client should be second param

Reply:
1d6837832a6f2353a4756420079142897404c7bf